src/test/rgw/rgw_multi/tests_ps.py

```diff
@@ -2736,6 +2745,7 @@ def test_ps_s3_multipart_on_master():
     assert_equal(len(events), 1)
     assert_equal(events[0]['Records'][0]['eventName'], 's3:ObjectCreated:CompleteMultipartUpload')
     assert_equal(events[0]['Records'][0]['s3']['configurationId'], notification_name+'_3')
+    print events[0]['Records'][0]['s3']['object']['size']
```

Comment:
this line introduced a bug, https://tracker.ceph.com/issues/45380, which was fixed by https://github.com/ceph/ceph/pull/34892

@Vicente-Cheng Could you please cherry-pick that PR here as well?

Reply:
@smithfarm done